### ISSUE TITLE
{Linked:Bug2235147}[Adaptive Cards> Designer]: The name of payment provider is not associated with the 'Open' control.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7729]
### Target Platforms

Other

### SDK Version

1.3

### Application Name

Adaptive cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25145.1011)
Browser: Edge Dev (105.0.1329.1 (Official build) dev (64-bit))
URL: https://adaptivecards.io/
Screen reader: Narrator

**Pre-Requisite:**
1. Attached the provided JSON in sample data editor and Card payload editor under Designer tab.
2. And activate the preview mode.

**Repro steps:**
1. Turn on the narrator.
2. Open the above URL in edge dev browser.
3. Navigate to the 'Open' button in card.
4. Observe the issue.

**Actual Result:**
'Stripe (Poc) i.e payment provider name is not associated with the 'Open' control.

**Expected Result:**
'Stripe (Poc) i.e payment provider name should be associated with the 'Open' control. Narrator should announce as **'Stripe (PoC) Open link'.**

**Observation:**
Same issue repro with NVDA on Chrome browser.

**User Impact:**
Screen reader user will get impacted as they will not be able to understand the proper information which in turn will not be able to track or navigate the content properly.

**MAS Reference link:**
[Info and Relationships - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01)

**"Have feedback to share on Bugs?** Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

https://user-images.githubusercontent.com/104483005/181704271-233b9e83-9fa8-4b23-b6ec-3a2381d9997b.mp4

### Card JSON

```JSON
{
  "type": "AdaptiveCard",
  "body": [
    {
      "items": [
        {
          "items": [
            {
              "color": null,
              "horizontalAlignment": null,
              "isSubtle": false,
              "maxLines": 0,
              "size": "medium",
              "text": "Payment providers",
              "weight": "bolder",
              "wrap": true,
              "separator": false,
              "type": "TextBlock"
            },
            {
              "color": null,
              "horizontalAlignment": null,
              "isSubtle": true,
              "maxLines": 0,
              "size": null,
              "text": "You'll view payments and take related actions straight from your provider accounts.",
              "weight": null,
              "wrap": true,
              "separator": false,
              "type": "TextBlock"
            }
          ],
          "separator": false,
          "type": "Container"
        },
        {
          "columns": [
            {
              "items": [
                {
                  "items": [
                    {
                      "columns": [
                        {
                          "width": "auto",
                          "items": [
                            {
                              "horizontalAlignment": null,
                              "size": "small",
                              "style": null,
                              "url": "https://us-prod.asyncgw.teams.microsoft.com/urlp/v1/url/content?url=https%3a%2f%2fwpforms.com%2fwp-content%2fuploads%2f2016%2f03%2faddon-icon-stripe.png",
                              "height": "auto",
                              "separator": false,
                              "type": "Image"
                            }
                          ],
                          "separator": false,
                          "type": "Column"
                        },
                        {
                          "width": "stretch",
                          "items": [
                            {
                              "color": null,
                              "horizontalAlignment": null,
                              "isSubtle": false,
                              "maxLines": 0,
                              "size": null,
                              "text": "**Stripe (PoC)**",
                              "weight": "bolder",
                              "wrap": true,
                              "separator": false,
                              "type": "TextBlock"
                            },
                            {
                              "color": null,
                              "horizontalAlignment": null,
                              "isSubtle": false,
                              "maxLines": 0,
                              "size": null,
                              "text": "Stripe, Inc.",
                              "weight": "lighter",
                              "wrap": true,
                              "spacing": "None",
                              "separator": false,
                              "type": "TextBlock"
                            }
                          ],
                          "separator": false,
                          "type": "Column"
                        }
                      ],
                      "separator": false,
                      "type": "ColumnSet"
                    },
                    {
                      "color": null,
                      "horizontalAlignment": null,
                      "isSubtle": false,
                      "maxLines": 0,
                      "size": null,
                      "text": "A complete payments platform, engineered for growth. Accepts payments with Stripe's powerful APIs and solutions which designed to help you capture",
                      "weight": null,
                      "wrap": true,
                      "separator": false,
                      "type": "TextBlock"
                    },
                    {
                      "color": null,
                      "horizontalAlignment": null,
                      "isSubtle": false,
                      "maxLines": 0,
                      "size": null,
                      "text": "",
                      "weight": null,
                      "wrap": false,
                      "separator": false,
                      "type": "TextBlock"
                    },
                    {
                      "actions": [
                        {
                          "url": "https://teams.microsoft.com/l/entity/74e1b13c-33cf-48e2-9471-850fdf9bcdb7/homeTab",
                          "title": "Open",
                          "type": "Action.OpenUrl"
                        }
                      ],
                      "separator": false,
                      "type": "ActionSet"
                    }
                  ],
                  "style": "default",
                  "separator": false,
                  "type": "Container"
                }
              ],
              "separator": false,
              "type": "Column"
            },
            {
              "items": [
                {
                  "items": [
                    {
                      "columns": [
                        {
                          "width": "auto",
                          "items": [
                            {
                              "horizontalAlignment": null,
                              "size": "small",
                              "style": null,
                              "url": "https://us-prod.asyncgw.teams.microsoft.com/urlp/v1/url/content?url=https%3a%2f%2fpmservices.cp.microsoft-int.com%2fstaticresourceservice%2fimages%2fv4%2flogo_stripe_small.png",
                              "height": "auto",
                              "separator": false,
                              "type": "Image"
                            }
                          ],
                          "separator": false,
                          "type": "Column"
                        },
                        {
                          "width": "stretch",
                          "items": [
                            {
                              "color": null,
                              "horizontalAlignment": null,
                              "isSubtle": false,
                              "maxLines": 0,
                              "size": null,
                              "text": "**Stripe**",
                              "weight": "bolder",
                              "wrap": true,
                              "separator": false,
                              "type": "TextBlock"
                            },
                            {
                              "color": null,
                              "horizontalAlignment": null,
                              "isSubtle": false,
                              "maxLines": 0,
                              "size": null,
                              "text": "Stripe Inc",
                              "weight": "lighter",
                              "wrap": true,
                              "spacing": "None",
                              "separator": false,
                              "type": "TextBlock"
                            }
                          ],
                          "separator": false,
                          "type": "Column"
                        }
                      ],
                      "separator": false,
                      "type": "ColumnSet"
                    },
                    {
                      "color": null,
                      "horizontalAlignment": null,
                      "isSubtle": false,
                      "maxLines": 0,
                      "size": null,
                      "text": "Stripe is a financial infrastructure platform for businesses. Millions of companies—from the world’s largest enterprises to the most ambitious startups—use Stripe to accept payments, grow their revenue, and accelerate new business opportunities.",
                      "weight": null,
                      "wrap": true,
                      "separator": false,
                      "type": "TextBlock"
                    },
                    {
                      "color": null,
                      "horizontalAlignment": null,
                      "isSubtle": false,
                      "maxLines": 0,
                      "size": null,
                      "text": "",
                      "weight": null,
                      "wrap": false,
                      "separator": false,
                      "type": "TextBlock"
                    },
                    {
                      "actions": [
                        {
                          "url": "https://pifd.cp.microsoft-int.com/v6.0/thirdpartyapi/payment-providers/stripe/link?pid=EAAAALVaK8nU4ZWefXeyQECGz8%2FVNKZr6ySr16%2Bq7OlNAI18GOcskwcGZjNjkHnOwkt%2BY3JZslgeKN8FbtKS5eIVtg%2Fm5KAvB5fCPvyiRGXBjt1tGygq615QDxaZ4c7Jd7JfoCwvdpV7G02YqHB2jC9xIq1C5%2BYQZ0A2A7nwT9%2B4K51zOPBjPvoWPwGGj4KKoZ0LQUJ3ItMknqRcyQYvGtbsj1RhHQomCqbxEyYQDI5Sd1O4&x-ms-correlation-id=hs3+6o2+AkedThG/.0",
                          "title": "Connect",
                          "type": "Action.OpenUrl"
                        }
                      ],
                      "separator": false,
                      "type": "ActionSet"
                    }
                  ],
                  "style": "default",
                  "separator": false,
                  "type": "Container"
                }
              ],
              "separator": false,
              "type": "Column"
            },
            {
              "items": [
                {
                  "items": [
                    {
                      "columns": [
                        {
                          "width": "auto",
                          "items": [
                            {
                              "horizontalAlignment": null,
                              "size": "small",
                              "style": null,
                              "url": "https://us-prod.asyncgw.teams.microsoft.com/urlp/v1/url/content?url=https%3a%2f%2fpmservices.cp.microsoft.com%2fstaticresourceservice%2fimages%2fv4%2flogo_paypal_rect.png",
                              "height": "auto",
                              "separator": false,
                              "type": "Image"
                            }
                          ],
                          "separator": false,
                          "type": "Column"
                        },
                        {
                          "width": "stretch",
                          "items": [
                            {
                              "color": null,
                              "horizontalAlignment": null,
                              "isSubtle": false,
                              "maxLines": 0,
                              "size": null,
                              "text": "**PayPal**",
                              "weight": "bolder",
                              "wrap": true,
                              "separator": false,
                              "type": "TextBlock"
                            },
                            {
                              "color": null,
                              "horizontalAlignment": null,
                              "isSubtle": false,
                              "maxLines": 0,
                              "size": null,
                              "text": "PayPal Holdings, Inc",
                              "weight": "lighter",
                              "wrap": true,
                              "spacing": "None",
                              "separator": false,
                              "type": "TextBlock"
                            }
                          ],
                          "separator": false,
                          "type": "Column"
                        }
                      ],
                      "separator": false,
                      "type": "ColumnSet"
                    },
                    {
                      "color": null,
                      "horizontalAlignment": null,
                      "isSubtle": false,
                      "maxLines": 0,
                      "size": null,
                      "text": "Accept payments online or in-store in over 100 currencies, get paid quickly, and let your customers pay the way they want with PayPal.",
                      "weight": null,
                      "wrap": true,
                      "separator": false,
                      "type": "TextBlock"
                    },
                    {
                      "color": null,
                      "horizontalAlignment": null,
                      "isSubtle": false,
                      "maxLines": 0,
                      "size": null,
                      "text": "",
                      "weight": null,
                      "wrap": false,
                      "separator": false,
                      "type": "TextBlock"
                    },
                    {
                      "actions": [
                        {
                          "url": "https://pifd.cp.microsoft-int.com/v6.0/thirdpartyapi/payment-providers/paypal/link?pid=EAAAAGTkHF0h%2Bt5UrDTkq8uMNDIdXpOkH12Pxs0a9b5Qw0Sig0AX7YNkomlkCjuf4FOy4ZLi7KEgGklX17wI2Wm%2BKuo8KLCDkRw5ZCaxYomEu6%2Bv%2Bp3nQITraxTWpWmNiJvsoV8nXbOp7TMHV1o1hSdEqOR%2FCYWjrPrBLWuSrVrn1RDzTfFUvm982YOr5umZG5n10sHCh%2BScB%2Fa02BY15hSPFngDUISasqiiSjtSIQtvPSDq&x-ms-correlation-id=hs3+6o2+AkedThG/.0",
                          "title": "Connect",
                          "type": "Action.OpenUrl"
                        }
                      ],
                      "separator": false,
                      "type": "ActionSet"
                    }
                  ],
                  "style": "default",
                  "separator": false,
                  "type": "Container"
                }
              ],
              "separator": false,
              "type": "Column"
            }
          ],
          "spacing": "small",
          "separator": false,
          "type": "ColumnSet"
        }
      ],
      "style": "emphasis",
      "bleed": true,
      "id": "providers",
      "separator": false,
      "type": "Container"
    }
  ],
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2",
  "actions": []
}
```

**Comment by Raisul123** [comment]
Closing the issue, As per dependent team comment 
![image](https://github.com/microsoft/AdaptiveCards/assets/23114460/bb30f590-b5bd-4b77-9b5d-6a87915b0cf4)
